VimeoUpload/Upload/Descriptor System/Cameo/CAMUploadDescriptor.swift

```diff
@@ -0,0 +1,356 @@
+//
+//  CAMUploadDescriptor.swift
+//  Cameo
+//
+//  Created by Westendorf, Michael on 6/27/16.
+//  Copyright © 2016 Vimeo. All rights reserved.
+//
+
+import Foundation
+import AFNetworking
+import VimeoNetworking
+
+public class CAMUploadDescriptor: ProgressDescriptor, VideoDescriptor
+{
+    let videoUrl: NSURL
+    let videoSettings: VideoSettings?
+    let thumbnailFileUrl: NSURL?
```

Comment:
is `thumbnailFileUrl` an actual file:// url? Can we make `videoUrl` and `thumbnailFileUrl` the same in that regard? As in drop the word `File` or add it?